packages/typespec-autorest/src/lib.ts

```diff
@@ -42,10 +42,17 @@ export interface AutorestEmitterOptions {
 
   /**
    * Directory where the examples are located.
-   * @default `{cwd}/examples`
```

Comment:
This wasn't actually true, if the examples directory wasn't set it wasn't loading examples at all.
I think it is better that we try to always load from `{project-root}/examples` as every azure service has this structure and would allow all tspconfig to be simplified by one config option